lib/Github/Api/AbstractApi.php

```diff
@@ -74,8 +75,11 @@ protected function get($path, array $parameters = array(), $requestHeaders = arr
             unset($parameters['ref']);
         }
         $response = $this->client->getHttpClient()->get($path, $parameters, $requestHeaders);
-
-        return ResponseMediator::getContent($response);
+        if ($response) {
```

Comment:
I think we should analyze this in detail. I suspect that this PR does not solve the issue. 

Why does it only happen for GET requests? If this was **the** solution, shouldn't it be applied for all HTTP methods?

Are you using caching when you get this error?